### PR TITLE
python: parse_uast_node should return Python instance

### LINKED ISSUE #352: Function parse_uast_node does not work as expected
## Expected Behavior
Using the `engine.parse_uast_node(uast)` turns a UAST bynary (obtained from the pyspark engine) into a python UAST node.

## Current Behavior
It returns a `class 'py4j.java_gateway.JavaObject` and its children can not be iterated or the members used the same way I do with nodes obtained with `babelfishclient.parse()`.

## Possible Solution
Using instead `Node.FromString(uasts.first()["uast"][0])` yields a node with type  `gopkg.in.bblfsh.sdk.v1.uast.generated_pb2.Node` with the expected behavriour.

## Steps to Reproduce (for bugs) 
```
from sourced.engine import Engine
from pyspark.sql import SparkSession

spark = SparkSession.builder\
.master("local[*]").appName("Examples")\
.getOrCreate()

engine = Engine(spark, "/repositories/siva/latest/*", "siva")
print("%d repositories successfully loaded" % (engine.repositories.count()/2))


uasts = engine.repositories.references.head_ref.commits.tree_entries.blobs\
    .classify_languages().where('lang = "Python"')\
    .extract_uasts().query_uast('//*[@roleIdentifier]')\
    .extract_tokens('result', 'tokens').select('uast')

uast = engine.parse_uast_node(uasts.first()["uast"][0])

for c in uast.children:
    print(c)
```

## Context
I was trying to get the binay UAST returned by the dataframe of the engine inside pyspark console into a usable python object (like the one that `babelfishclient.parse()` ) returns.

## Your Environment (for bugs)

* Version used: 0.5.5
* Operating System and version: linux mint 18
* Some needed resources to reproduce the problem: pyspark console (2.2.1) python (3.5.2)

### PR DESCRIPTION
**Pull request by erizocosmico**
Fixes #352 

**NOTE:** that this package will need `libxml-dev` now to work (because `bblfsh` client python needs it)

### CODE REVIEW

**Comment by erizocosmico** [comment]
UPDATE: client-python is Python 3 only. Should we proceed with this, then? Removing support for 2.x seems like a big thing to remove.

**Comment by smola** [comment]
[Python 2 is dead](https://pythonclock.org/), but we should check with others company-wide.

File: .travis.yml [hunk 1]
```diff
@@ -44,9 +44,26 @@ matrix:
     - jdk: openjdk8
 
     - language: python
-      python: 2.7
+      python: 3.4
 
       install:
+        - sudo apt install libxml2-dev
```
Comment (jfontan) [inline .travis.yml]: I may be wrong but I believe package installation will be a little bit faster executing only one `apt install` command:

```
sudo apt install libxml2-dev curl build-essential
```

**Comment by smola** [comment]
Confirmed that we can drop Python 2 support if needed.

**Pull request merged**